runtime/src/main/scala/akka/grpc/internal/ClientState.scala

```diff
@@ -6,54 +6,37 @@ package akka.grpc.internal
 
 import java.util.concurrent.CompletionStage
 
+import scala.concurrent.Future
+
 import akka.Done
 import akka.actor.ClassicActorSystemProvider
-import akka.annotation.InternalApi
-import akka.annotation.InternalStableApi
+import akka.annotation.{ InternalApi, InternalStableApi }
 import akka.event.LoggingAdapter
-import akka.grpc.GrpcClientSettings
-import akka.grpc.scaladsl.Grpc
-
-import scala.compat.java8.FutureConverters._
-import scala.concurrent.Future
+import akka.grpc.{ GrpcChannel, GrpcClientSettings }
 
 /**
  * INTERNAL API
  *
- * Client utilities taking care of Channel reconnection and Channel lifecycle in general.
+ * Deprecated: This class wraps a GrpcChannel for compatibility with clients generated by previous versions.
  */
+@deprecated("Kept for binary compatibility between generated code and runtime", "akka-grpc 2.1.4")
```

Comment:
Is a deprecation annotation appropriate here?
Should version number be 2.1.4 (next patch release) or 2.2.0 (next minor release)?
Should version number include library name? I see both styles being used elsewhere in the project.

Reply:
> Is a deprecation annotation appropriate here?

Yes, I think so :+1: 

> Should version number be 2.1.4 (next patch release) or 2.2.0 (next minor release)?

I think the next patch release would be best

> Should version number include library name? I see both styles being used elsewhere in the project.

I see https://www.scala-lang.org/api/current/scala/deprecated.html promotes adding the library name, but TBH I don't think it's necessary - I'm OK with either